guildwars2/commerce.py

```diff
@@ -165,40 +204,39 @@ class CommerceMixin:
             colour=await self.get_embed_color(ctx))
         data.set_author(
             name='Delivery overview of {0}'.format(doc["account_name"]))
-        data.set_thumbnail(url="https://wiki.guildwars2.com/"
-                           "images/thumb/d/df/Black-Lion-Logo.png"
-                           "/300px-Black-Lion-Logo.png")
-        data.set_footer(text="Black Lion Trading Company")
+        data.set_footer(
+            text="Black Lion Trading Company",
+            icon_url="https://api.gw2bot.info/resources/icons/bltc.png")
         coins = results["coins"]
         items = results["items"]
-        items = items[:20]  # Get only first 20 entries
-        item_quantity = []
         itemlist = []
         if coins == 0:
             gold = "Currently no coins for pickup."
         else:
             gold = self.gold_to_coins(ctx, coins)
-        data.add_field(name="Coins", value=gold, inline=False)
-        counter = 0
+        data.add_field(name="> **Gold**", value=gold, inline=False)
         if len(items) != 0:
+            length_counter = 0
             for item in items:
-                item_quantity.append(item["count"])
                 itemdoc = await self.fetch_item(item["id"])
-                itemlist.append(itemdoc)
-            for item in itemlist:
-                item_name = item["name"]
-                # Get quantity of items
-                quantity = item_quantity[counter]
-                counter += 1
-                data.add_field(
-                    name=item_name,
-                    value="x {0}".format(quantity),
-                    inline=False)
+                line = "**{}** {}".format(item["count"], itemdoc["name"])
+                if length_counter + len(line) < 6000:
+                    length_counter += len(line)
+                    itemlist.append(line)
+                else:
+                    break
+            # Split list for formatting into half
+            items_left = itemlist[:math.ceil(len(items) / 2)]
+            items_right = itemlist[math.ceil(len(items) / 2):]
+            items_right.insert(0, zero_width_space)
+            data = embed_list_lines(
+                data, items_left, "> **ITEMS**", inline=True)
+            data = embed_list_lines(
+                data, items_right, zero_width_space, inline=True)
         else:
             if coins == 0:
                 return await ctx.send("Your delivery box is empty!")
```

Comment:
I think the delivery box should always be displayed, even if it's empty, like in game.